app/models/partners/partner.rb

```diff
@@ -105,6 +105,171 @@ def verified?
+          income_verification: income_verification,
+          internal_db: internal_db,
+          maac: maac,
+          enthnic_composition: {
+            african_american: population_black,
+            caucasian: population_white,
+            hispanic: population_hispanic,
+            asian: population_asian,
+            american_indain: population_american_indian,
+            pacific_island: population_island,
+            multi_racial: population_multi_racial,
+            other: population_other
+          },
+          zip_codes_served: zips_served,
+          poverty_information: {
+            at_fpl_or_below: at_fpl_or_below,
+            above_1_2_times_fpl: above_1_2_times_fpl,
+            greater_2_times_fpl: greater_2_times_fpl,
+            poverty_unknown: poverty_unknown
+          },
+          ages_served: ages_served
+        },
+        executive_director: {
+          name: executive_director_name,
+          phone: executive_director_phone,
+          email: executive_director_email
+        },
+        contact_person: {
+          name: program_contact_name,
+          phone: program_contact_phone,
+          mobile: program_contact_mobile,
+          email: program_contact_email
+        },
+        pick_up_person: {
+          method: pick_up_method,
+          name: pick_up_name,
+          phone: pick_up_phone,
+          email: pick_up_email
+        },
+        distribution: {
+          distribution_times: distribution_times,
+          new_client_times: new_client_times,
+          more_docs_required: more_docs_required
+        },
+        funding: {
+          source: sources_of_funding,
+          diapers: sources_of_diapers,
+          budget: diaper_budget,
+          diaper_funding_source: diaper_funding_source
+        },
+        documents: []
+      }
+    end
+
+    #
+    # Creates the neccessary ActiveStorage records to be able to
+    # fetch files uploaded on Partnerbase (legacy seperate app).
+    # This is due to the fact that ActiveStorage does not support
+    # multiple databases and is not built to be able to fetch
+    # files from the `partner_*` databases.
+    #
+    # NOTE: This should be deprecated or removed once Partnerbase
+    # users are uploading files directly in the partnerbase app
+    # hosted within diaperbase.
+    #
+    def sync_attachments_from_partnerbase!
+      ActiveRecord::Base.transaction do
+        attachment_records = Partners::Base.connection.execute(
+          <<-SQL
+            SELECT asb.*, asa.* filename FROM active_storage_blobs asb
+              LEFT JOIN active_storage_attachments asa ON asa.blob_id = asb.id
+              WHERE record_type = 'Partner'
+              AND record_id = '#{id}'
+          SQL
+        )
+
+        old_acs_attachments = ActiveStorage::Attachment.where(
+          record_type: self.class.name,
+          record_id: id
+        )
+        old_blob_ids = old_acs_attachments.map(&:blob_id)
```

Comment:
```suggestion
        old_blob_ids = old_acs_attachments.pluck(:blob_id)
```
minor, but I think pluck is more performant here